scripts/db/postgres/dev/upgrade_dev_database.sh

```diff
@@ -26,13 +25,12 @@ pushd "${ZEN_SRC_ROOT}" &> /dev/null
 # NOTE(stephen): Using the simplified base app since we don't need to import all
 # the extra crap that is needed to actually start the full flask server. We only
 # need the core.
-export DATABASE_URL="postgresql://postgres:@localhost/${DB_NAME}"
+export DATABASE_URL="postgresql://postgres:@${DATABASE_HOST:-localhost}/${DB_NAME}"
```

Comment:
Yep - see comments above re. trusted - I added it in - but really - like I said before - I think the whole thing needs a re-factor. It's get's very confusing having multiple places with defaults, and multiple ways of constructing DATABASE_URL

Reply:
I'm hesitant to start doing a re-factor like that as part of this PR though!

Reply:
Yeah, it's changes that would touch a lot of code and require testing all of those to be sure it works. Also, in order to not make the next Harmony release a pain, we would need to then go update all of those files in our repo.

Ideally what should happen is we don't construct the postgres url anywhere in our code and everything uses `DATABASE_URL`. That currently isn't done because there's no docker compose to set it and I'm not running `DATABASE_URL=postgresql://postgres:zenysis@postgres:5432/br-local yarn br-server` every time I run the web server.

Instead of doing that refactor now, what would you think about changing the values in the docker compose to align with what's hardcoded so it all still works? `POSTGRES_PASSWORD` -> nothing, `POSTGRES_DB` -> `localhost`, and `DATABASE_URL` -> `postgresql://postgres:@localhost:5432/${ZEN_ENV}-local`.